home/dot_config/task/Taskfile.yml

```diff
@@ -149,7 +155,10 @@ tasks:
       - task: profile:backup
         vars:
           AppName: Google Chrome
-          ProfileFolder: ~/Library/Application Support/Google/Chrome/Default
+          ProfileFolderDarwin: ~/Library/Application Support/Google/Chrome/Default
+          ProfileFolderLinux: ~/.config/google-chrome/default
```

Comment:
We don't need the folders stored in this location. You can remove this file from the PR.

Maybe we should store the Windows PATHs to save to a new, unsupported section in each of the mackup configuration files. i.e. `[windows_localappdata_files]` section could contain the `Google/Chrome/User Data/Default` path